src/gcp/cloudscheduler.ts

```diff
@@ -86,7 +87,7 @@ export function updateJob(job: Job): Promise<any> {
  * @throws { FirebaseError } if an error response other than 404 is received on the GET call.
  */
 export async function createOrReplaceJob(job: Job): Promise<any> {
-  const jobName = `${job.name.split("/").slice(-1)}`;
+  const jobName = `${job.name.split("/").pop()}`;
```

Comment:
i definitely do want pop here since its clearer, but weirdly enough, there is no behavioral change here. I was doing some testing since I was confused how this was working, but apparently, if you pass an array into a`${}`, it concatenates the contents of the array into a comma separated string. Since slice(-1) returns a length one array, this ends up with the same behavior.
![Screen Shot 2019-05-20 at 2 49 15 PM](https://user-images.githubusercontent.com/4635763/58054152-873a1500-7b0e-11e9-8d56-d0ae3a004315.png)

Reply:
Scary. Glad we have a better solution now.

That said, since `job.name` is a string, do we _really_ need to wrap this in a template string (backticks) as well?

Reply:
Because of the behavior-changing aspect of the backticks, I would really prefer we don't use them.

Reply:
Very good point, im not sure why this was even ever wrapped in template string